buildSrc/src/main/kotlin/multiplatform-common.gradle.kts

```
@@ -0,0 +1,78 @@
+@file:Suppress("UNUSED_VARIABLE")
+
+import org.jetbrains.dokka.gradle.DokkaTask
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.dokka")
+    `maven-publish`
```

```suggestion
    id("maven-publish")
```
Keep that consistent

---

It would be `id("org.gradle.maven-publish")`

We aren't using that yet so I'm going to remove it in general.